css/style.css

```diff
@@ -2,7 +2,142 @@
  * Add your custom styles below
  * 
  * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
+ * - Be organized, use comments and separate your styles into meaningful chunks
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+html {
+  font-family: "Mukta", sans-serif;
```

Comment:
Choosing a custom font here and declaring it at the HTML level for consistency makes all the difference to the aesthetics of your site, really good decision.